code/game/objects/items/devices/flashlight.dm

```diff
@@ -327,7 +327,7 @@
 
 /obj/item/device/flashlight/flare/Initialize()
 	. = ..()
-	fuel = rand(9.5 MINUTES, 10.5 MINUTES)
+	fuel = 15 MINUTES
```

Comment:
if this is no longer random, you can just put it in the definition of /obj/item/device/flashlight/flare, with the name and desc and stuff